UKHO.UIAutomationFramework/UKHO.SeleniumWebDriver/SeleniumElement.cs

```diff
@@ -75,6 +76,8 @@ public void DoubleClick()
                 action.DoubleClick(element);
```

Comment:
Could... you add this into a method called "performClickWorkaroundForXProblem()" - this then documents why it's being done without adding comments.